disconnect-when-backgrounded.js

```diff
@@ -1,18 +1,30 @@
 var disconnectTimer = null;
 
+// 10 seconds by default
+var disconnectTime = (Meteor.settings && Meteor.settings.public && Meteor.settings.public.disconnectTime || 10) * 1000;
```

Comment:
I will make the syntax change. Or we could just go with ms instead of seconds.

Meteor settings aren't really supposed to be used that way. Maybe I'll make it clearer in the docs. But the settings.json file looks something like this:
```
{
  'public': {
    'disconnectTimeSec': 10
  }
}
```